src/kfs_lib/includes/kfs_common.h

```diff
@@ -19,9 +19,13 @@
 char *MOUNT_PATH;
 char *LOCAL_DISC_CACHE_PATH;
 char *LOG_CONFIG_PATH;
+#define COPYMODE        0644
```

Comment:
macros make sense when you change it to different builds, don't put it everywhere, use constants instead